pkg/bufwriter/writer.go

```diff
@@ -0,0 +1,28 @@
+// Package bufwriter provides a wrapper for io.WriteCloser that uses
+// buffered output via a bufio.Writer and calls Flush on close.
+// Not clear why bufio.Writer doesn't do this.
+package bufwriter
+
+import (
+	"bufio"
+	"io"
+)
+
+type Writer struct {
+	io.WriteCloser
```

Comment:
Doing this will ensure that all calls to the `Writer.Write` will not hit the `bufio.Writer`... I do not think this is want you want.

Reply:
What was I thinking ?!